flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java

```diff
@@ -34,8 +36,28 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.rules.TemporaryFolder;
 
-public abstract class FlinkTestBase extends AbstractTestBase {
+public abstract class FlinkTestBase extends TestBaseUtils {
+
+  private static final int DEFAULT_PARALLELISM = 4;
+
+  private static final org.apache.flink.configuration.Configuration config =
```

Comment:
Nit: How about just import the `org.apache.flink.configuration.Configuration` class and use: 

```java
private static final Configuration config = new Configuration().set(CoreOptions.CHECK_LEAKED_CLASSLOADER, false);
```
here.

Reply:
will fix